### ISSUE TITLE
elements to elements

### ISSUE THREAD

**Issue by krober10nd** [CHLNDDEV/oceanmesh#14]
* This connectivity is required for the mesh cleaning step at the end. However, if there are disjoint/disconnected portions of the mesh my approach doesn't work...Any suggestions would be appreciated.

https://github.com/CHLNDDEV/oceanmesh/blob/c59f0393eeb449fcdaaa70dd1c0e66c754d0775c/oceanmesh/clean.py#L22

```
def _cell_to_cell(t):
    """Cell to cell connectivity table.
        Cell `i` is connected to cells `ctoc[ix[i]:ix[i+1]]`
        By connected, I mean shares a mutual edge.
    Parameters
    ----------
    t: array-like
        Mesh connectivity table.
    Returns
    -------
    ctoc: array-like
        Cell numbers connected to cells.
    ix: array-like
        indices into `ctoc`
    """
    nt = len(t)
    t = np.sort(t, axis=1)
    # NB: we use order="F" to reshape because the `np.tile` command below
    e = t[:, [[0, 1], [0, 2], [1, 2]]].reshape((nt * 3, 2), order="F")
    trinum = np.tile(np.arange(nt), 3)
    j = _arg_sortrows(e)
    e = e[j, :]
    trinum = trinum[j]
    k = np.argwhere(~np.diff(e, axis=0).any(axis=1))
    ctoc = np.concatenate((trinum[k], trinum[k + 1]), axis=1)
    ctoc = np.append(ctoc, np.fliplr(ctoc), axis=0)
    ctoc = ctoc[np.argsort(ctoc[:, 0]), :]
    idx = np.argwhere(np.diff(ctoc[:, 0])) + 1
    idx = np.insert(idx, 0, 0)
    idx = np.append(idx, len(ctoc))
    return ctoc, idx
```

**Comment by krober10nd** [comment]
Upon more thought, best to just insert the vertices into a delaunay2 triangulation object and iterate around the connected faces returning them back to Python.

**Comment by krober10nd** [comment]
addressed in #13
